examples/Thermostat/README.md

```diff
@@ -0,0 +1,87 @@
+# Thermostat
+
+This larger example presents an application with a client (think app running on your phone) that interacts with a device
```

Comment:
```suggestion
This more extensive example presents an application with a client (think app running on your phone) that interacts with a device
```

I think large almost have a bit of a negative context, I think more extensive is a better word choice.

Reply:
Removed larger.